public/static/js/projects.js

```diff
@@ -220,6 +220,9 @@ mciModule.controller('ProjectCtrl', function($scope, $window, $http, $location,
           item.pr_testing_enabled = false;
           item.commit_queue.enabled = false;
           item.enabled = false;
+          item.subscriptions = _.filter($scope.subscriptions, function(d) {
```

Comment:
Don't you want to copy all subscriptions? Are you trying to exclude those that were added without saving?

Reply:
Yes, this excludes those that were added without saving.